client/my-sites/stats/stats-purchase/stats-purchase-shared.tsx

```diff
@@ -112,8 +113,16 @@ const StatsSingleItemPagePurchaseFrame = ( {
 	children,
 	isFree = false,
 }: StatsSingleItemPagePurchaseFrameProps ) => {
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+
 	return (
-		<div className={ classNames( COMPONENT_CLASS_NAME, `${ COMPONENT_CLASS_NAME }--single` ) }>
+		<div
+			className={ classNames(
+				COMPONENT_CLASS_NAME,
+				`${ COMPONENT_CLASS_NAME }--single`,
+				! isOdysseyStats ? `${ COMPONENT_CLASS_NAME }--is-wpcom` : ''
```

Comment:
Might be cleaner to understand if we put it this way:

```suggestion
				{ `${ COMPONENT_CLASS_NAME }--is-wpcom` : isOdysseyStats }
```

Reply:
Putting this class on the entire page would be more versatile.
`index.tsx` in the same directory and line 156 `<div className={ classNames( 'stats', 'stats-purchase-page' ) }>`

Reply:
> Might be cleaner to understand if we put it this way:

I tried it first but got errors, and I am still trying to figure out why. cc @kangzj 

<img width="740" alt="截圖 2023-09-05 下午5 00 37" src="https://github.com/Automattic/wp-calypso/assets/6869813/0a90df77-d6a6-44df-8c85-f893699cccb2">

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/81359/commits/22732765dda07d8a3081eb45ba35c66a110cfebd.

Reply:
Ah sorry I think I missed the brackets, it'll work this way:

```
{ [`${ COMPONENT_CLASS_NAME }--is-wpcom`]: isOdysseyStats }
```

Reply:
Thanks for the hint! The brackets make sense. 👍🏼